submissions/adeyemiJuwon/css/style.css

```diff
@@ -0,0 +1,285 @@
+
+   *{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    font-size: 16px;
+    line-height: 18px;
+}
+/**********************************
+  HEADING
+  ***********************************/
+
+header{
+    display: flex;
+    justify-content: flex-end;
+    background-color: rgb(255, 166, 95);
+    color:black;
+    align-items: center;
+
+}
+ .logo{
+    margin-right:auto;
+}
+.navbar-link{
+    list-style: none;
+}
+.navbar-link li {
+    display: inline-block;
+    line-height: 19px;
```

Comment:
You can remove the other line-height in the file since you have added it to the body

Reply:
i can"t remove it because they have different line-height. i can only remove the one that have the same as the body